scripts/transform-genuary/in.json

```diff
@@ -0,0 +1,108 @@
+    "1055689244468854834": "28 Generative poetry",
+    "1055689304338346064": "29 Maximalism",
+    "1055689379592548393": "30 Minimalism",
+    "1055689602561757184": "31 Deliberately break one of your previous images",
+    "1056111498910511135": "Reference"
+  },
+  "authors": {
+    "1011054407560994876": {
+      "name": "sama#0386",
+      "id": "1011054407560994876",
+      "avatar": "https://cdn.discordapp.com/avatars/1011054407560994876/99fef939828ef64ce3d6b97e156927c7.png"
+    },
+    "260901399729012736": {
+      "name": "EthanThatOneKid#3456",
+      "id": "260901399729012736",
+      "avatar": "https://cdn.discordapp.com/avatars/260901399729012736/35bc87750c2f4ab1886b6ed8a199654a.png"
+    },
+    "268909280160120833": {
+      "name": "Angus#7980",
+      "id": "268909280160120833",
+      "avatar": "https://cdn.discordapp.com/avatars/268909280160120833/5da1c7373baed3acab4a65c9b62b45ac.png"
+    },
+    "268934031989538817": {
+      "name": "부미#8525",
+      "id": "268934031989538817",
+      "avatar": "https://cdn.discordapp.com/avatars/268934031989538817/e6097080e7905565eff4a92729152193.png"
+    },
+    "305390566693404673": {
+      "name": "ToxicPoison#0656",
+      "id": "305390566693404673",
+      "avatar": "https://cdn.discordapp.com/avatars/305390566693404673/dfd6db411264f7ea5955b10343526698.png"
+    },
+    "316384186573520898": {
+      "name": "jiink#6249",
+      "id": "316384186573520898",
+      "avatar": "https://cdn.discordapp.com/avatars/316384186573520898/d7391adeb19bcf31cf0f1ea05e87d294.png"
+    },
+    "619385101473677312": {
+      "name": "HallowsYves#4728",
+      "id": "619385101473677312",
+      "avatar": "https://cdn.discordapp.com/avatars/619385101473677312/40d1d6945c943681751edbbd9fd55492.png"
+    },
+    "869320557672423455": {
+      "name": "Karbas#0001",
+      "id": "869320557672423455",
+      "avatar": "https://cdn.discordapp.com/avatars/869320557672423455/6316de9f278e12541d9f2676b9e1c708.png"
+    }
+  },
+  "pins": [
+    {
+      "channel_id": "1055680245740605450",
+      "timestamp": "2023-01-08T05:22:30.491Z",
+      "author_id": "268934031989538817",
+      "text": "I wanted to do like pokemon esc cards but with Harry Potter. I didnt have time to finish up the weird bug (when you flip the card its like flipped with it LOL). Too lazy to do other cards.",
+      "attachments": [
+        {
+          "id": "1061515218653548564",
+          "filename": "Screen_Recording_2023-01-07_at_9.20.04_PM.mov",
+          "content_type": "video/quicktime",
+          "size": 29223341,
+          "url": "https://cdn.discordapp.com/attachments/1055680245740605450/1061515218653548564/Screen_Recording_2023-01-07_at_9.20.04_PM.mov",
+          "proxy_url": "https://media.discordapp.net/attachments/1055680245740605450/1061515218653548564/Screen_Recording_2023-01-07_at_9.20.04_PM.mov",
+          "height": 1454,
+          "width": 2222
+        }
+      ]
+    },
+
```

Comment:
Nit: Rogue empty line.

Reply:
Resolved in <https://github.com/EthanThatOneKid/acmcsuf.com/pull/821/commits/034a277d3a450c1ef240c226f96fb4029c5c4375>.